lib/wasi/Cargo.toml

```diff
@@ -49,8 +49,31 @@ tracing-wasm = "0.2"
 default = ["sys-default"]
 
 sys = ["wasmer/sys"]
-sys-default = ["wasmer/sys-default", "sys", "logging", "host-fs", "sys-poll", "host-vnet" ]
+sys-default = [
+    "wasmer-min-features",
+    "compiler-cranelift", 
+    "sys", 
+    "logging", 
+    "host-fs", 
+    "sys-poll", 
+    "host-vnet"
+]
```

Comment:
this does not solve the issue, we want wasmer-wasi **not** to depend on a compiler by default.

Reply:
Alright, but then the tests will fail, I'd have to adjust the `make test`. It would just be cumbersome for the user. That's why I did it this way.